make_queue/static/make_queue/js/make_reservation.js

```diff
@@ -1,7 +1,7 @@
 let reservations = [];
 let reservationRules = [];
 let reservationCalendarDate = new Date();
-var canIgnoreRules = false;
+let canIgnoreRules = false;
```

Comment:
This `let` will collide with the `canIgnoreRules` variable in the `reservation_calendar.html` file. (Both are used on the new reservation page)

Reply:
Hm, it seems to me like this variable is always overwritten by the one in `calendar.html` (assuming that's what you meant with `reservation_calendar.html`, as `calendar.html` is included in it), but correct me if I'm wrong. I think it would be more easily understandable if we either remove it from `make_reservation.js` or write something like `var canIgnoreRules; // Defined in calendar.html`.

Reply:
The variable is probably overwritten in all cases, as I think this code was written before the calendar.html code. I like you current solution.